app/controllers/cards_controller.rb

```diff
@@ -44,6 +58,6 @@ def update
 
   private
     def card_params
-      return params.permit(:title, :content, :image_url, :board_id)
+      return params.permit(:text, :emoji, :id)
```

Comment:
Very minor, but I'm not sure there's any value to including `:id` in the permitted list.

AFAICT it would only allow people to attempt to force a particular ID value when creating a new card, or change an ID when updating an existing card, neither of which are behavior we want to encourage I think.